src/dialects/abstract/query-interface.js

```diff
@@ -277,7 +277,14 @@ export class QueryInterface {
     options = { ...options };
     options.cascade = options.cascade || options.force || false;
 
-    const sql = this.queryGenerator.dropTableQuery(tableName, options);
+    let sql;
+
+    // TODO: this should be refactored to probably use DialectSupports
+    if (this.queryGenerator.dialect.name === 'postgres') {
+      sql = this.queryGenerator.dropTableQuery(tableName, options);
+    } else {
+      sql = this.queryGenerator.dropTableQuery(tableName);
+    }
```

Comment:
So this is messy and definitely should not be merged like this, but I'm unsure what the proper course of action is. Add something like `dialect.supports.drop.cascade`?
And what should we do with `DROP_TABLE_QUERY_SUPPORTABLE_OPTIONS`? Remove that and incorporate `dialect.supports.drop` into `rejectInvalidOptions`? Or keep both?

Reply:
I vote for `dialect.supports.drop.cascade`

Reply:
You can do this:

```ts
options.cascade = options.cascade != null ? options.cascade
  : (options.force && dialect.supports.drop.cascade) ? true
  : undefined;
```

that way cascade is only set if it's supported

Reply:
I think it should be fixed in https://github.com/sequelize/sequelize/pull/15235/commits/7d3099ddf03d961c8c5fe311d3a72a2d89c5db9e